layouts/contact/single.html

```diff
@@ -69,7 +69,7 @@ <h5 class="lg:ml-6 text-xl text-white font-bold">
                             <label for="checkbox"><span class="ml-2 text-white" id="consentText">{{ .subscribe_consent_text }}</span></label>
                         </div>
                         <div class="md:flex items-center justify-between mt-6 text-center md:mt-7 lg:mt-8">
-                            <button id="contact-bttn" type="submit" data-sitekey="{{ .Site.Params.recaptchaKey }}" data-action="contact" class="flex mr-9 bg-white mt-6 px-7 py-4 rounded-lg hover:bg-white/90">{{.submit }}</button>
+                            <button id="contact-bttn" type="submit" data-sitekey="{{ site.Params.recaptchaKey }}" data-action="contact" class="flex mr-9 bg-white mt-6 px-7 py-4 rounded-lg hover:bg-white/90">{{.submit }}</button>
```

Comment:
Hugo recommends using `site`: https://gohugo.io/functions/global/site/

There are some places where we've used `.Site` but don't see a need to update since we haven't run into any issues.